### PR TITLE
add search bar Airbnb

### PR DESCRIPTION
**Pull request by VvynnykV**
- [DEMO LINK](https://VvynnykV.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://VvynnykV.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Let's improve and try to pass the tests.

- there shouldn't be any horizontal scroll

image

- looks like your input doesn't fit the form. stylize the input. form should only to it's size
image

**Comment by VvynnykV** [comment]
I know that input fitting its my problem, but I don't know yet how to fix it, maybe give me some small advise)

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
```
Comment (VitaliyBondarenko1982) [inline src/style.css:14]: for fixing horizontal scroll try to add such styles
```suggestion

  * {
    box-sizing: border-box;
  }
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com/"
+      method="get"
+      data-qa="big"
+      class="bigField field"
```
Comment (VitaliyBondarenko1982) [inline src/index.html:18]:
```suggestion
      class=" field"
```
Screenshot from 2023-03-10 13-24-13
Screenshot from 2023-03-10 13-23-18

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**